src/Stock.js

```diff
@@ -13,6 +13,13 @@ class Stock extends React.Component {
             stockTimeValues: [],
             stockPriceValues: [],
             stockVolumeValues: [],
+            companyName: [],
+            company_52WeekHigh: [],
+            company_52WeekLow: [],
+            companyDescription: [],
+            priceToSalesRatio: [],
+            profitMargin: [],
+            RevenuePerShareTTM: [],
```

Comment:
can you make Revenue have a lower case r for consistency (and change it in the other variable with the same issue too)

Reply:
Consistency issue has been fixed!